package-lock.json

```diff
@@ -10,6 +10,8 @@
       "hasInstallScript": true,
       "license": "ISC",
       "dependencies": {
+        "@dvsa/cvs-type-definitions": "4.0.0",
+        "@types/express-serve-static-core": "4.17.20",
```

Comment:
When I install this package out locally this dependency is in devDependencies, not dependencies. After the rebase may be worth double checking we are also committing the updated package-lock.